thirdparty/patchmatchnet/models/patchmatch.py

```diff
@@ -593,20 +598,24 @@ def forward(
         """Forward method for PatchMatch
 
         Args:
-            ref_feature: feature from reference view
-            src_features: features from source views
-            ref_proj: projection matrix of reference view
-            src_projs: source matrices of source views
-            depth_min: minimum virtual depth
-            depth_max: maximum virtual depth
-            depth: current depth map
-            img: image
-            view_weights: list to store weights from source views
+            ref_feature: feature from reference view, (B, C, H, W)
+            src_features: features from (Nview-1) source views, (Nview-1) * (B, C, H, W), where Nview is the number of
+                input images (or views) of PatchmatchNet
+            ref_proj: projection matrix of reference view, (B, 4, 4)
+            src_projs: source matrices of source views, (Nview-1) * (B, 4, 4), where Nview is the number of input
+                images (or views) of PatchmatchNet
+            depth_min: minimum virtual depth, (B,)
+            depth_max: maximum virtual depth, (B,)
+            depth: current depth map, (B,1,H,W) or None
+            img: image, (B,C,image_H,image_W)

```

Comment:
is image_H, image_W not equal to H,W ?

Reply:
yes, feature shape (H, W) might be 1/2, 1/4 or 1/8 of the (image_H, image_W) in different stages